courses/free-courses-it.md

```diff
@@ -170,6 +172,7 @@
 
 ### Programmazione
 
+* [Corso C per sistemi Embedded](https://www.youtube.com/watch?v=XpMF2pnEWVw) - Deid LAB  
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator.

Reply:
Thanks, I'll fix this :)